### DISCUSSION TITLE
召喚主のペットモンスターが消滅すると召喚されたモンスターも消滅する仕様の拡張

### DISCUSSION THREAD

**Discussion by habu1010** [hengband/hengband#318] [Ideas]
ペットが召喚したモンスターは、召喚主がなんらかの理由で消滅した場合は連鎖的にすべて消える。
これは現在ペットのみの仕様だが、データ構造的には通常モンスターにも適用できる。

- 手下を召喚するが本体を倒したら手下も消えるモンスター
- 集団で現れるが親玉を倒したらすべて消えるモンスター

等の特徴を持つモンスターが実装できそう。

**Comment by RRyukkuriRyu** [comment]
消えるというのは撃破とは別扱いでしょうか？例えば、手下がナズグルのような準ユニークで本体が破壊で消滅した時、次回出現時は手下が出てこないということにはならないということですか？

**Reply by habu1010** [reply]
撃破とは別扱いですね。
それこそ破壊で消したのと同じなので、出現数限定モンスターがでなくなるという事もないですし、思い出の撃破数が増えることもないです。
